driver/ppm_events.c

```diff
@@ -626,11 +640,17 @@ int val_to_ring(struct event_filler_arguments *args, uint64_t val, u32 val_len,
 			/*
 			 * Handle NULL pointers
 			 */
+#ifdef WDIG
+			strcpy(args->buffer + args->arg_data_offset,
+				"(NULL)");
+			len = (int)strlen(args->buffer + args->arg_data_offset);
+#else
 			len = strlcpy(args->buffer + args->arg_data_offset,
 				"(NULL)",
 				max_arg_size);
+#endif
```

Comment:
Can we provide an implementation of strlcpy for Windows instead? Or maybe just a `ppm_strlcpy` that does the right thing for both OSes

Reply:
done